From 37e5db63f83c1ea13016a4b4d628eaf98d1d4298 Mon Sep 17 00:00:00 2001
Message-Id: <37e5db63f83c1ea13016a4b4d628eaf98d1d4298.1423872525.git.feitong.yi@intel.com>
In-Reply-To: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
References: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
From: Gaurav K Singh <gaurav.k.singh@intel.com>
Date: Wed, 21 Jan 2015 22:23:18 +0530
Subject: [PATCH 59/95] MUST_REBASE [VPG]: drivers/video/adf: Add MIPI Hactive
 reg offset

MIPI HORIZ_ACTIVE_AREA_COUNT is not getting initialized with a correct
offset causing pipe programming failures. This issue is visible only in
dual display mode, as in such cases, GOP doesn't program MIPI registers.

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

For: GMINL-5348
Change-Id: I6cd2975dd941dceaaba2c25a9a184fb78761d34a
Signed-off-by: Gaurav K Singh <gaurav.k.singh@intel.com>
---
 drivers/video/adf/intel/core/vlv/vlv_dsi_port.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/adf/intel/core/vlv/vlv_dsi_port.c b/drivers/video/adf/intel/core/vlv/vlv_dsi_port.c
index 2f4e685..536cf94 100644
--- a/drivers/video/adf/intel/core/vlv/vlv_dsi_port.c
+++ b/drivers/video/adf/intel/core/vlv/vlv_dsi_port.c
@@ -452,6 +452,7 @@ bool vlv_dsi_port_init(struct vlv_dsi_port *port, enum port enum_port,
 	port->ctrl_offset = MIPI_CTRL(enum_port);
 	port->dphy_param_offset = MIPI_DPHY_PARAM(enum_port);
 	port->dpi_res_offset = MIPI_DPI_RESOLUTION(enum_port);
+	port->hactive_count_offset = MIPI_HACTIVE_AREA_COUNT(enum_port);
 	port->hfp_count_offset = MIPI_HFP_COUNT(enum_port);
 	port->hsync_count_offset = MIPI_HSYNC_PADDING_COUNT(enum_port);
 	port->hbp_count_offset = MIPI_HBP_COUNT(enum_port);
-- 
1.7.9.5

